### PR TITLE
Clean-ups and bugfixes, stage 2

### PR DESCRIPTION
**Pull request by favonia**
Yet another collection of clean-ups.

### CODE REVIEW

**Comment by jonsterling** [comment]
Looks good. Feel free to merge at will whenever you're ready.

File: src/redprl/lcf_model.fun [hunk 1]
```diff
@@ -8,7 +8,7 @@ struct
   type tactic = Lcf.jdg Lcf.tactic nominal
   type multitactic = Lcf.jdg Lcf.multitactic nominal
   type env = multitactic Syn.VarCtx.dict
-  exception InvalidRule
+  (* exception InvalidRule *) (* commented out as unused *)
```
Comment (jonsterling) [inline src/redprl/lcf_model.fun]: feel free to delete this

File: src/redprl/lcf.sml [hunk 2]
```diff
@@ -70,10 +69,10 @@ struct
         {doc = PP.empty, env = Metavar.Ctx.empty, idx = 0}
     end
 
-  fun prettyValidation env vld =
+  fun prettyValidation _(*env*) vld =
     let
       open RedPrlAbt infix \
-      val (us,xs) \ m = outb vld
+      val _(*(us,xs)*) \ m = outb vld
```
Comment (jonsterling) [inline src/redprl/lcf.sml]: feel free to write `val _ \ m = ...`

**Comment by favonia** [comment]
I moved the fix that hopefully implements the hcom for paths from #206. Let's merge this huge collection.

**Comment by favonia** [comment]
@jonsterling Sorry I realized some rebasing did not seem to work. Let me fix them first.

File: src/redprl/lcf_syntax.fun [hunk 3]
```diff
@@ -13,7 +13,7 @@ struct
   type multitactic = abt
   type sign = Sig.sign
   type ann = Pos.t * string option
-  type state = Sig.jdg Lcf.state
+  (* type state = Sig.jdg Lcf.state *) (* commented out as unused *)
```
Comment (favonia) [inline src/redprl/lcf_syntax.fun]: Should we remove this line too?

Comment (jonsterling) [inline src/redprl/lcf_syntax.fun]: Let's delete

**Pull request merged**